_articles/how-to-contribute.md

```diff
@@ -216,6 +216,8 @@ Open source isn't an exclusive club; it's made by people just like you. "Open so
 
 You might scan a README and find a broken link or a typo. Or you're a new user and you noticed something is broken, or an issue that you think should really be in the documentation. Instead of ignoring it and moving on, or asking someone else to fix it, see whether you can help out by pitching in. That's what open source is all about!
 
+Good first issue is a feature GitHub ships for encouraging new and beginner open source contributors for contributing in an open source project. If you are new for an open sourcr project or have never contributed in an any open source projects you can write a fix for a good first issues. These tickets helps you learn the open source organisation culture, maintainers ethics and communication practices. The maintainers keep the thought in mind that you are a new and beginner contributor in open source project if you are taking a good first issue.
```

Comment:
I'd suggest a much tighter paragraph and probably talk about `/contribute` as mentioned in https://github.com/github/opensource.guide/issues/1052 as the previous paragraph is about looking for stuff to work on in a specific repo.

Reply:
Again I agree with Mike. In addition, this paragraph should go after the quotation. See my suggestion below.